Resources/Prototypes/Entities/Objects/Devices/cartridges.yml

```diff
@@ -17,57 +26,44 @@
   - type: NotekeeperCartridge
 
 - type: entity
-  parent: BaseItem
+  parent: BaseCartridge
   id: NewsReadCartridge
   name: News Cartridge
   description: A program for reading news
   components:
-  - type: Sprite
-    sprite: Objects/Devices/cartridge.rsi
-    state: cart-y
   - type: UIFragment
     ui: !type:NewsReadUi
   - type: Cartridge
     programName: news-read-program-name
     icon:
-      sprite: Interface/Misc/program_icons.rsi
       state: news_read
```

Comment:
i dont think overriding just the `state` field like this and getting `sprite` from the parent is going to work since its nested. is this tested?

Reply:
> is this tested?

no